features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/sso/logout/template.jag

```diff
@@ -37,8 +37,7 @@
                 if(samlLogoutRequest==null){
                         //logout request failed because there is no session details in anymore. Here trying to do a login request.
                         if (log.isDebugEnabled()) {
-                            log.debug("logout request failed because there is no session details in
-                                                    SAMLSSORelyingPartyObject. Here it is trying to do a login request");
+                            log.debug("logout request failed because there is no session details in SAMLSSORelyingPartyObject. Here it is trying to do a login request");
```

Comment:
Can we break into two statement so that, line length won't exceed 120 characters?